examples/project-tracker/actix_server.rs

```diff
@@ -11,16 +11,16 @@ use actix_web::{web, App, HttpResponse, HttpServer};
 use juniper::http::GraphQLRequest;
 use std::collections::HashMap;
 
-use super::{GlobalContext, ReqContext};
-use warpgrapher::Engine;
+use super::{AppGlobalContext, AppRequestContext};
```

Comment:
This would probably be handled under a separate issue, but what would you think about pulling the project-tracker example out to a separate repo altogether.  Now that we've coverted from server to engine, this seems to pull in a lot of irrelevant dependencies.

Reply:
Agreed. I'll open up a ticket for this.

Reply:
https://github.com/warpforge/warpgrapher/issues/41